<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>meshDialog</class>
 <widget class="QWidget" name="meshDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>532</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>532</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>532</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>meshDialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#meshDialog{
background-color: rgb(39, 39, 39);
}

  QCheckBox {      
	color: rgb(219, 219, 219);
  }

QCheckBox::indicator:unchecked {
 
	
	image: url(:/images/check_normal.png);
}
QCheckBox::indicator:checked {
  
	image: url(:/images/check_checked.png);
}

QPushButton
{
color: rgb(255, 255, 255);

border-image: url(:/images/btn_normal_normal.png) 2;
border-width: 2;
}

QPushButton:hover
{
border-image: url(:/images/btn_normal_hover.png) 2;
border-width: 2; 
}

QPushButton:pressed
{
border-image: url(:/images/btn_normal_press.png) 2;
border-width: 2;
}

QPushButton:disabled
{
color: rgb(102, 102, 102);
border-image: url(:/images/btn_normal_disabled.png) 2;
border-width: 2;
}
QRadioButton {    
	color: #9b9b9b;	
    
	font: 14pt &quot;Arial&quot;;
    
}

QRadioButton::indicator:unchecked {
    
	color: #ffffff;
    width: 16px;
    height:16px;
	border-image: url(:/images/ridio_norml.png);
}
QRadioButton::indicator:checked {
   color: #ffffff;
 width: 16px;
    height:16px;
	border-image: url(:/images/rido_checked.png);
}

QLineEdit {
     border-width: 1px;
     border-style: solid;
     border-color: #606060;    
	color: rgb(231, 231, 231);
    padding-left:10 px;
   
	background-color: rgb(255, 255, 255,0);

 }

QLineEdit:disabled

{

	 border-width: 1px;
     border-style: solid;
     border-color: #606060;    
	color: rgb(102, 102, 102);

}
QLabel{
	
	color: rgb(231, 231, 231);
    
	font: 16px &quot;Arial&quot;;
}

QSpinBox{
	
	background-color: #00606060;
border: 1px solid #606060;
   
	color: #DBDBDB;
    padding-left:5px
   
}

QSpinBox::up-button
{
   padding:1px;
	
	background-image: url(:/images/up_normal.png);
    height: 10px;
    width:12px;
}

QSpinBox::up-button:pressed
{
   padding:1px;
	background-image: url(:/images/up_active.png);
    height: 10px;
    width:12px;
}
QSpinBox::down-button{
    
	
	background-image: url(:/images/down_normal.png);
   height: 10px;
    width:12px;
  
   padding:1px;
}
QSpinBox::down-button:pressed{
    
	background-image: url(:/images/down_active.png);
   height: 10px;
    width:12px;
  
   padding:1px;
}

QSlider::groove:horizontal 

{       
      border: 1px solid #606060;  
      height: 8px;                        
      border-radius: 4px;
	  margin: 0px 0; 
      left: 4px; right: 4px;
 }                                                          

QSlider::handle:horizontal 
{                       

	
	
	border-image: url(:/images/slider_handle.png);

     width: 22px;                                           

     margin: -7px -4px -7px -4px;                           

 }                                                          

                                                            

QSlider::sub-page:horizontal

{                               

                    
	 border: 1px solid #606060;  
      height: 6px;                        
      border-radius: 3px;

	
		background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #2E91ED,  stop:0.5 #5BA8EF, stop:1 #2E91ED);

}                                                           

QSlider::add-page:horizontal

{                            

	 border: 1px solid #606060;  
      height: 6px;                        
      border-radius: 3px;

     
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #272727, stop:1 #1C1C1C);
		

}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QWidget" name="widget_back" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>500</width>
            <height>532</height>
           </size>
          </property>
          <property name="text">
           <string>MeshType</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <spacer name="horizontalSpacer_24">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_UnWatertight">
          <property name="text">
           <string>UnWatertight</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>17</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_HalfWatertight">
          <property name="text">
           <string>HalfWatertight</string>
          </property>
          <property name="iconSize">
           <size>
            <width>18</width>
            <height>18</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>17</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_Watertight">
          <property name="text">
           <string>Watertight</string>
          </property>
          <property name="iconSize">
           <size>
            <width>18</width>
            <height>18</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_25">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>17</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QWidget" name="modelquality_radioButtons" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <spacer name="horizontalSpacer_26">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>ModelQuality</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_8">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButton_high">
           <property name="text">
            <string>High</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_9">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>17</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButton_mid">
           <property name="text">
            <string>Mid</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_10">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>17</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButton_low">
           <property name="text">
            <string>Low</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_11">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <spacer name="horizontalSpacer_20">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_removeSpike">
          <property name="text">
           <string>Remove Spike</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_markerHoleFilling">
          <property name="text">
           <string>Marker Hole Filling</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_21">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>97</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <spacer name="horizontalSpacer_12">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="minimumSize">
           <size>
            <width>105</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Simplification</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="horizontalSlider_simpli">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox_simpli">
          <property name="minimumSize">
           <size>
            <width>55</width>
            <height>28</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_13">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>65</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <spacer name="horizontalSpacer_14">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="minimumSize">
           <size>
            <width>105</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Filter</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="horizontalSlider_optimize">
          <property name="maximum">
           <number>3</number>
          </property>
          <property name="pageStep">
           <number>1</number>
          </property>
          <property name="value">
           <number>1</number>
          </property>
          <property name="sliderPosition">
           <number>1</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox_optimize">
          <property name="minimumSize">
           <size>
            <width>55</width>
            <height>28</height>
           </size>
          </property>
          <property name="maximum">
           <number>3</number>
          </property>
          <property name="value">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_15">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>65</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <spacer name="horizontalSpacer_18">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="minimumSize">
           <size>
            <width>105</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Smooth</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="horizontalSlider_smooth">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>3</number>
          </property>
          <property name="pageStep">
           <number>1</number>
          </property>
          <property name="value">
           <number>1</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox_smooth">
          <property name="minimumSize">
           <size>
            <width>55</width>
            <height>28</height>
           </size>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>3</number>
          </property>
          <property name="value">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_19">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>65</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <spacer name="horizontalSpacer_16">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="minimumSize">
           <size>
            <width>105</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Remove Small 
Floating Parts</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="horizontalSlider_removeSmallParts">
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="value">
           <number>1</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox_removeSmallParts">
          <property name="minimumSize">
           <size>
            <width>55</width>
            <height>28</height>
           </size>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="value">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_17">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>65</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <item>
         <spacer name="horizontalSpacer_22">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_apply">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>44</height>
           </size>
          </property>
          <property name="text">
           <string>Apply</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_revert">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>44</height>
           </size>
          </property>
          <property name="text">
           <string>Revert</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_confirm">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>44</height>
           </size>
          </property>
          <property name="text">
           <string>Confirm</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_23">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>horizontalSlider_simpli</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_simpli</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>246</x>
     <y>201</y>
    </hint>
    <hint type="destinationlabel">
     <x>391</x>
     <y>207</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_simpli</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_simpli</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>394</x>
     <y>192</y>
    </hint>
    <hint type="destinationlabel">
     <x>314</x>
     <y>200</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_optimize</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_optimize</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>291</x>
     <y>259</y>
    </hint>
    <hint type="destinationlabel">
     <x>408</x>
     <y>261</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_optimize</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_optimize</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>395</x>
     <y>249</y>
    </hint>
    <hint type="destinationlabel">
     <x>331</x>
     <y>257</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_removeSmallParts</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_removeSmallParts</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>236</x>
     <y>316</y>
    </hint>
    <hint type="destinationlabel">
     <x>418</x>
     <y>334</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_removeSmallParts</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_removeSmallParts</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>394</x>
     <y>317</y>
    </hint>
    <hint type="destinationlabel">
     <x>323</x>
     <y>317</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_smooth</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_smooth</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>321</x>
     <y>390</y>
    </hint>
    <hint type="destinationlabel">
     <x>398</x>
     <y>390</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_smooth</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_smooth</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>393</x>
     <y>374</y>
    </hint>
    <hint type="destinationlabel">
     <x>332</x>
     <y>374</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
